frontend/src/images/icons/ModelEvaluationIcon.ts

```diff
@@ -0,0 +1,13 @@
+import { createIcon } from '@patternfly/react-icons/dist/esm/createIcon';
+
+const ModelEvaluationIcon = createIcon({
+  name: 'ModelEvaluationIcon',
+  width: 40,
+  height: 40,
```

Comment:
@ikeola13 just a small suggestion, can we make the width and height 36? referring the other icons currently present